src/transformers/configuration_lxmert.py

```diff
@@ -176,3 +176,8 @@ def __init__(
         self.visual_feat_loss = visual_feat_loss
         self.output_hidden_states = output_hidden_states
         self.output_attentions = self.output_attentions
+        self.num_hidden_layers = {
+            "vision": r_layers,
+            "cross_encoder": x_layers,
+            "language": l_layers
+        }
```

Comment:
This is very different to what we usually do, but as it's the first multi-transformer architecture we can have a bit more freedom. I think this approach makes sense, but we'll have to discuss with other team members before the final merge in `master` if we decide to merge this.